tests/cypress/e2e/teacher_page/class_page/add_student_copy_link.cy.js

```diff
@@ -0,0 +1,22 @@
+import {loginForTeacher, logout} from '../../tools/login/login.js'
+import { createClass} from '../../tools/classes/class'
+describe('Is able to see teacher page', () => {
+  it('Passes', () => {
+    
+    loginForTeacher();
+    cy.wait(500);
+    
+
+    cy.get('#class_view_button').click();
+
+    cy.get('.green-btn').contains("Add students").click();
```

Comment:
This will only work when the site is set to English, can you make it more generic?